applications/crossbar/src/crossbar_view.erl

```diff
@@ -991,12 +1070,21 @@ last_key(_LastKey, JObjs, Limit, Returned) when Returned == Limit ->
 %% @end
 %%------------------------------------------------------------------------------
 -spec format_response(load_params()) -> cb_context:context().
-format_response(#{total_queried := TotalQueried
-                 ,queried_jobjs := JObjs
-                 ,context := Context
-                 ,last_key := NextStartKey
-                 ,start_key := StartKey
-                 }) ->
+format_response(#{context := Context}=LoadMap) ->
+    case cb_context:resp_status(Context) of
```

Comment:
Is there a situation it couldn't be `success` at this point?

Reply:
Any error from kz_datamgr will be propagated in `Context` within the `LoadMap`.